FredLin.md

```diff
@@ -1,39 +1,61 @@
 [Open bugs assigned to me](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=assignee%3Agasolin%40mozilla.com) (ASSIGNED = current working on; NEW = backlog)
 
-## 8/8 ~ 8/12
+## 8/15 ~ 8/19
+
+Attend Coscup at 8/20~21.
+Present about webby project, Slide https://hackmd.io/s/HJ7dfgVc#/
 
 Devtools
 
 - tab motion design (PLAN)
-  - helen (UX) sad she most wanted it than box-model view (Bug 1150496)
-  - tabbar redesign bugs https://bugzilla.mozilla.org/buglist.cgi?quicksearch=%5Bdevtools-toolbar%5D&list_id=13146931
+  - Helen (UX) sad she most wanted it than box-model view (Bug 1150496)
+  - Review UI specs at Bug 1292054 - Polish sidebar style and add tab motion
+    - found inspector, new debugger(on github), network monitor(xul) have sidebar tabs in 3 different implementation and places, so its not the work that should be done at this time.
+  - hold study group to check how toolbox tab works, blocked by Bug 1245921 - Change toolbox tabs to HTML, taken by :miker
+  - Therefore we told :pbro & :honza we will improve box-model view (Bug 1150496) first
```

Comment:
Thanks for the update. Bug 1150496 itself looks big. Could you update me on the plan also?